README.md

```diff
@@ -1,107 +1,210 @@
-<!-- deno-fmt-ignore-file -->
 # deno2node
 
-Transpiles Deno projects into `.js` and `.d.ts` for Node.js.
+Compile your [Deno] project to run on [Node.js].
 
-Uses [`ts-morph`] to rewrite imports, typecheck, and emit.
+## CLI Usage from Deno
 
-## Motivation
+No installation needed. Simply `cd` into the directory of your project, and run:
+
+```sh
+deno run --no-check --allow-read=. --allow-write=. \
+    https://deno.land/x/deno2node/src/cli.ts <tsConfigFilePath>
+```
 
-Writing libraries Deno-first
-makes it easy to publish to https://deno.land/x,
-and simplifies development experience:
+You need to substitute `<tsConfigFilePath>` by a path to your `tsconfig.json`
+file. `deno2node` passes it on to the shipped `tsc` for compiling your code.
 
-> Deno \[...\] requires no explicit transpilation step,
-> and ships with 0conf tooling that works well together.
+## CLI Usage from Node.js
 
-## CLI Usage
+As a by-product of end-to-end testing, a Node.js build is also available:
 
 ```sh
-$ deno run \
-  --no-check \
-  --unstable  \
-  --allow-read \
-  --allow-write=<outDir> \
-  https://deno.land/x/deno2node/src/cli.ts \
-  <tsConfigFilePath>
+npm install --save-dev --save-prefix='~' deno2node
 ```
 
-<!-- deno-fmt-ignore -->
-As a by-product of end-to-end testing,
-Node.js build is also available:
+Now add a script to `package.json` so you can run it with `npm run build`:
 
-```sh
-$ npm install --save-dev --save-prefix='~' deno2node
+```jsonc
+{
+  // yada yada ...
+  "scripts": {
+    "build": "deno2node <tsConfigFilePath>"
+  }
+}
 ```
 
+Instead of adding a script, you can also run it directly:
+
 ```sh
-$ deno2node <tsConfigFilePath>
+npx deno2node <tsConfigFilePath>
 ```
 
-`tsconfig.json` is used to specify `compilerOptions` and source `files` to `include`.
+## Motivation
+
+Writing code for Deno is fun. It's so much simpler.
 
-[API reference] explains transformations and configuration.
+If you have tried it, you know what we mean.
 
-Note: output and diagnostics will change across minor versions.
+If not, here is how you develop for Deno: Don't wait for TypeScript to
+transpile. Just run it. Don't wait for unit tests. They are truly fast. Linting?
+It is always instant. Zero configuration files. The tooling is standardised,
+built-in, and works well together. No `package.json` to tediously fill out. No
+more “did you try `rm -rf node_modules`?” Everything just works out of the box.
+What a relief.
+
+Try it, and you will know what we mean.
+
+## How it Works
+
+There are three main steps to this.
+
+1. Transform the code base. This means:
+   - Rewrite all import statements, e.g. to handle explicit vs. implicit file
+     extensions
+   - Reverse npm package transformations from `skypack.dev`, `esm.sh`,
+     `deno.land/std`, and `nest.land/std`.
```

Comment:
```suggestion
     extensions.
   - Use bare specifiers in `std/node`, https://esm.sh and https://skypack.dev imports.
```

Reply:
You can modify this PR as you wish, or not. Apply the changes you'd like to see—I do not wish to start discussions about this :)

Reply:
Ok, although I'd like to ensure that I don't mess up your flair :)

I'll commit this, tweak a few more things, wait for your review, and then finally merge.

Reply:
I'm glad you think it has flair :D
Don't worry.

I can do the review once it's fine :)